app/services/process_state.rb

```diff
@@ -34,6 +85,14 @@ def state
     end
   end
 
+  def self.i18n_namespace
+    'candidate_flow_'
+  end
+
+  def self.state_count(_)
```

Comment:
Will be implemented with https://github.com/DFE-Digital/apply-for-postgraduate-teacher-training/pull/1220.